core/src/main/java/org/jruby/Ruby.java

```diff
@@ -1717,7 +1717,6 @@ private void initBuiltins() {
         addLazyBuiltin("java.rb", "java", "org.jruby.javasupport.Java");
         addLazyBuiltin("jruby.rb", "jruby", "org.jruby.ext.jruby.JRubyLibrary");
         addLazyBuiltin("jruby/util.rb", "jruby/util", "org.jruby.ext.jruby.JRubyUtilLibrary");
-        addLazyBuiltin("jruby/type.rb", "jruby/type", "org.jruby.ext.jruby.JRubyTypeLibrary");
```

Comment:
`org.jruby.ext.jruby.JRubyTypeLibrary` class was gone for quite some time - as explained in the commit msg